app/models/concerns/agent/feature_flags.rb

```diff
@@ -0,0 +1,29 @@
+module Agent::FeatureFlags
+  extend ActiveSupport::Concern
+
+  AVAILABLE_FEATURES = %w[new_planning].freeze
+
+  def feature_enabled?(feature)
+    feature_flags && feature_flags[feature] == true
+  end
+
+  def enable_feature(feature)
```

Comment:
nitpick : dans l’API de cette fonction je ne trouve ça pas super intuitif si ça va sauvegarder en db ou non

proposition : 

```suggestion
  def enable_feature(feature, persist: false)
```

Reply:
En vrai pour le moment, on n'a pas de cas où on ne persiste pas
Peut-être que je peux remonter le `update!` ici et renommer en `enable_feature!` 
Qu’en penses-tu ?

Reply:
D’ailleurs, je peux passer ça en privé vu que ce n’est utilisé que dans le toggle pour le moment

Reply:
👍 yay en privé, avec ou sans bang tout me va, ce sont des remarques vraiment bonus !